content/en/docs/reference/controllers/daemonset.md

```diff
@@ -0,0 +1,102 @@
+---
+toc_hide: true
+title: DaemonSet controller
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+The DaemonSet {{< glossary_tooltip term_id="controller" text="controller" >}} manages
+{{< glossary_tooltip term_id="pod" text="Pods">}} configured via a
+{{< glossary_tooltip term_id="daemonset" >}} object, to make sure that some
+(or all) {{< glossary_tooltip term_id="node" text="Nodes">}} run a copy of a
+Pod.
```

Comment:
Maybe something to the effect of:

`..to make sure that all Nodes meeting some user-specified condition run a copy of a Pod.`

I guess there are two main concepts we should convey:
1) That the user can define a set of conditions so that one Pod runs on every node meeting that condition
2) By default the DaemonSet pod will run on every node